phone/src/locale/en.json

```diff
@@ -192,6 +192,7 @@
     "APPS_SETTINGS_OPTION_NOTIFICATIONVOL": "Notification Volume",
     "APPS_SETTING_STREAMER_MODE_DESC": "Hide images before they are shown",
     "APPS_SETTINGS_SET_CUSTOM_WALLPAPER": "Set new wallpaper",
+    "APPS_SETTINGS_INVALID_WALLPAPER": "Not a valid image or link!",
```

Comment:
If we r gonna keep with this locale style should I revert my nested key format?

Reply:
Ehh...I can rather change it. Nested looks better.